source/git/git_package_release_guide.rst

```diff
@@ -0,0 +1,71 @@
+
+Preamble
+~~~~~~~~
+
+The release version of the package is created as a stable self-contained version of the package (called a *Release*), tagging it with a version and creating accompanying metadata such as Release Notes and Changelogs. Depending on the development environment, Changelogs and version tags can be automated. Release notes are targeted at the end user, highlighting the features of the new release, whereas a Changelog is usually a detailed description of all changes since the last release version.
+
+The standard convention for naming versions (also known as `Semantic Versioning <https://semver.org/>`__) is to tag a release as `MAJOR.MINOR.PATCH<alpha/beta>` where:
+
+1. `MAJOR` refers to an update to the whole package that is backwards incompatible with the previous versions.
+2. `MINOR` refers to either new features, functional updates, or a large number of bundled bug fixes.
+3. `PATCH` refers to small bug fixes, hotfixes, or minor changes in the code. 
+4. Optional tag `<alpha/beta>` (angular braces `<>` indicate optional keywords) - This indicates the package has been released but is currently being tested and may not be stable. `alpha` testing usually implies testing within the broader community of the team that developed the package, whereas `beta` testing implies a wider release to the audience/end users, who can then report bugs which can be fixed. 
+
+On github, creating a release usually entails creating a stable version of the package on a branch, then creating a release tag for the specific stable commit. These branches can be standard main/develop or a dedicated release branch, which is usually dependent on the conventions of the organization creating the package. 
+
+When numbering versions, in rare cases, a package may need to be released before it is tested, or even when it is in active development. These should be tagged as a `v0.MINOR.PATCH<alpha>` release and it is good practice to highlight that the release is unstable or in active development. Once a package has a first stable release, the leading digit should roll over to `v1.MINOR.PATCH<alpha>`. For all subsequent releases, the first digit should only roll over once backward compatibility is broken, so subsequent tags should go as `v1.0.0`, `v1.1.0`, `v1.2.0`, etc... 
+
+Pre-release, release branches and archiving releases
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Note that the tagging of `v0.MINOR.PATCH<alpha>` is not the same as creating a Pre-release on github, as tagging a commit is not the same as creating a release. UASAL organization preference is to *NOT create Pre-release versions* of packages.
```

Comment:
```suggestion
Note that the tagging of `v0.MINOR.PATCH<alpha>` is not the same as creating a Pre-release on github, as tagging a commit is not the same as creating a release. UASAL organization preference is to *NOT utilize the Pre-release* functionality in GitHub. This is primarily to simplify the release process. This may be revisited in the future, especially if software with a large user base develops.
```